setup.py

```diff
@@ -52,7 +52,7 @@ def configuration(parent_package='', top_path=None):
                        'License :: OSI Approved :: BSD License',
                        'Topic :: Scientific/Engineering'],
           configuration=configuration,
-          install_requires=['numpy', 'scipy'],
```

Comment:
Should put a minimum version on matplotlib that provides the required functionality.